app/upgrades/v2/upgrades.go

```diff
@@ -33,6 +35,21 @@ func CreateUpgradeHandler(mm *module.Manager, configurator module.Configurator,
 			return nil, err
 		}
 
+		// update reward distribution
+		err = UpdateRewardTrackerEventLastProcessedHeight(ctx, keepers.IncentiveKeeper)
+		if err != nil {
+			return nil, err
+		}
+
 		return migrations, nil
 	}
 }
+
+// UpdateRewardTrackerEventLastProcessedHeight sets the current block height -1 to the reward tracker
+// so when a new BTC block is rewarded it doesn't need to go through all the missing block heights
+// since block 0.
```

Comment:
```suggestion
// UpdateRewardTrackerEventLastProcessedHeight sets the current block height - 1 to the reward tracker
// so that BTC reward distribution starts from the height at which the upgrade happens
```

Am I understanding it correctly?